decidim-system/app/views/decidim/system/organizations/edit.html.erb

```diff
@@ -14,7 +14,7 @@
 
   <div class="field">
     <%= f.label :available_authorizations %>
-    <%= f.collection_check_boxes :available_authorizations, Decidim.authorization_handlers, :name, :name %>
+    <%= f.collection_check_boxes :available_authorizations, Decidim.authorization_handlers, :itself, :itself %>
```

Comment:
`itself`?????????????? OMG I didn't know this existed 😭

Reply:
Pretty crazy, eh? Ruby style. I remember seeing it in docs or somewhere, but only now I found a use case.

Reply:
It's awesome ❤️

Reply:
Sad & nice at the same time!